spring-cloud-dataflow-server-local/src/main/resources/META-INF/kafka-apps.properties

```diff
@@ -37,3 +37,8 @@ sink.tcp=maven://org.springframework.cloud.stream.app:tcp-sink-kafka:@registry.a
 sink.throughput=maven://org.springframework.cloud.stream.app:throughput-sink-kafka:@registry.app.defaultVersion@
 sink.websocket=maven://org.springframework.cloud.stream.app:websocket-sink-kafka:@registry.app.defaultVersion@
 task.timestamp=maven://org.springframework.cloud.task.app:timestamp-task:@registry.app.defaultVersion@
```

Comment:
Not that they use binder the same way as stream applications, but the Task applications do publish events to brokers as well. While I like the idea of keeping `task` applications in a separate file, at the same time, this is going to add one extra step for users to register them. Also, this experience is only for the Local server, so I'd vote for keeping them in the same file.